packages/SideNavigation/Link/__tests__/__snapshots__/Link.spec.jsx.snap

```diff
@@ -1,20 +1,16 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SideNavigation.Link renders 1`] = `
-<a
-  className="link"
+<styled.a
```

Comment:
Could you please change `shallow` rendering to `mount` for snapshots so that it will include css generated by `styled-components`.

Reply:
Done